src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -267,7 +267,7 @@ internal async Task<CloudBlockBlob> GetHashBlobAsync()
         public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
```

Comment:
Note: Since GetSyncTriggersPayload() doesn't expose the new sortFunctionsMetadata param, triggers will be sorted for both background synctriggers and the explicit api call even though the background task is the one the issue was originally opened for.